test/library/standard/FileSystem/lydia/chmod/sub_test

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+#
+# Set umask as expected by at least some of these tests.
+#
+umask 0022 && ${CHPL_HOME}/util/test/sub_test $1
```

Comment:
@lydia-duncan can you review this change?

Reply:
Looks good to me!  Thank god I commented that on the test in question XD

Reply:
That comment made it so easy to figure out why those tests were failing for me!